dockerfiles/che/entrypoint.sh

```diff
@@ -306,11 +306,10 @@ add_cert_to_truststore() {
   echo "$1" > $SELF_SIGNED_CERT
 
   # make sure that owner has permissions to write and other groups have permissions to read
-  # note that this might not work when running as anyuid OpenShift user, which is why we're forcing a true if it fails
-  chmod 644 $JAVA_TRUST_STORE || true
+  chmod 644 $JAVA_TRUST_STORE
```

Comment:
you have removed the `|| true` here, and the comment explaining why it's needed on openshift. Why?

Reply:
It is not needed any more, because now current user is owner of the file.